tests/unit/select_connection_ioloop_tests.py

```diff
@@ -10,22 +10,33 @@
 
 import logging
 try:
+    #pylint: disable=F0401
```

Comment:
I understand that pylint error/warning suppression is effective only in the block it appears.
And I think it's ok or desirable to ignore warnings about failed imports after a failed import.